aws/table_aws_route53_resolver_endpoint.go

```diff
@@ -0,0 +1,315 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/service/route53resolver"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableAwsRoute53ResolverEndpoint(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_route53_resolver_endpoint",
+		Description: "AWS Route53 Resolver Endpoint",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("id"),
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFoundException"}),
+			Hydrate:           getAwsRoute53ResolverEndpoint,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsRoute53ResolverEndpoint,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name that you assigned to the Resolver endpoint when you submitted a CreateResolverEndpoint.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "The ID of the Resolver endpoint.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags assigned to the Resolver endpoint.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsRoute53ResolverEndpointTags,
+				Transform:   transform.FromField("Tags"),
+			},
```

Comment:
We should try to keep all the JSON fields at end of the table, please change accordingly